file_upload.rb

```diff
@@ -110,49 +110,53 @@ def upload_file(url, headers, file_path)
 # was previously uploaded to Sentera's cloud storage with one
 # of the mutations in Sentera's GraphQL API that accepts a
 # file ID as an input. In this example, we'll use the
-# import_files GraphQL mutation, and attach the file to
-# a file owner that belongs to the caller's FieldAgent organization.
+# import_feature_set GraphQL mutation to attach the file to
+# a feature set owned by the specified file owner
 #
 # @param [string] file_id ID of the uploaded file
-# @param [string] file_owner_type Type of file owner to create. For example,
-#                                 FEATURE_SET, MOSAIC, etc.
-# @param [string] file_owner_sentera_id Sentera ID of the resource
-#                                       (field, survey, feature set, etc.)
-#                                       to which the file should be attached.
+# @param [string] owner_type Type of owner that will own the
+#                            feature set. For example: SURVEY.
+# @param [string] owner_sentera_id Sentera ID of the resource
+#                           (field, survey, feature set, etc.)
+#                           that will own the feature set that
+#                           is created.
 #
 # @return [Hash] Hash containing results of the GraphQL request
 #
-def use_file(file_id, file_owner_type, file_owner_sentera_id)
+def use_file(file_id, owner_type, owner_sentera_id)
```

Comment:
```suggestion
def import_feature_set(file_id, owner_type, owner_sentera_id)
```

Might as well matched the embedded GQL mutation, eh?